### ISSUE TITLE
WCAG 4.1.2 test 15: false positive

### ISSUE THREAD

**Issue by jzuidweg** [veraPDF/veraPDF-wcag-algs#95]
[This document](https://github.com/veraPDF/veraPDF-wcag-algs/files/7396910/test-document.pdf) fails **WCAG 4.1.2 test 15** ('_Paragraph is incorrectly tagged as a numbered heading_'). The error applies to a line ('Kanttekeningen' on page 4) that is a correctly tagged heading. This is a false positive.
